src/types/tasks.ts

```diff
@@ -27,7 +27,7 @@ export interface TaskTypes {
   created_at: string
   updated_at: string
   document_type: DocumentType
-  documents: Document[]
+  document: Document[]
```

Comment:
I've changed `documents` to `document`, because API was giving `document`, but I believe `documents` is the correct one